pnut.c

```diff
@@ -2016,8 +2019,13 @@ void get_tok() {
 #endif
 
 
+#ifdef __gcc__
```

Comment:
It would be more portable to have:

```
#ifndef __FILE__
#define __FILE__ "<unknown>"
#endif

#ifndef __LINE__
#define __LINE__ (-1)
#endif
...
```